### DISCUSSION TITLE
Source of the Riivolution banner music

### DISCUSSION THREAD

**Discussion by rollerozxa** [AerialX/rawksd#11] [General]
I'm opening this issue to see if I could get any closer to finding out the source of the Riivolution banner music, the 13 second loop that plays on the installable Wii menu channel for Riivolution.

(For reference, here's the music extracted from the banner file, converted to .wav: [riivolution_banner_music.wav.gz](https://github.com/AerialX/rawksd/files/12927921/riivolution_banner_music.wav.gz))

My first instinct was to upload it onto a music recognition site, which detected a match in... [some sort of romantic audiobook (mild NSFW warning)](https://www.youtube.com/watch?v=tPubBn8RUfA). If the year of the audiobook being published (2002) is to be trusted, it would mean that it originates from some freely available source and not specifically made for Riivolution.

Any help or pointers from the original developers for tracking down the source of the music would be appreciated, if there are any.

**Comment by AerialX** [comment]
Hah, nice find!
I unfortunately don't have many details for you but I believe it was @Tempus who designed the banner way back then.
